windrose/windrose.py

```diff
@@ -0,0 +1,133 @@
+import numpy as np
+import pandas as pd
+
+
+class WindroseAnalysis:
+    def __init__(self, ref_n=90, ref_e=0):
+        self.north = ref_n
+        self.east = ref_e
+
+    @staticmethod
+    def calculate_wind_frequency(ndirections, vwinds, speed, dirc, direction):
+        """
+        Calculate the frequency of wind speed and direction.
+
+        Args:
+            ndirections (int): Number of direction bins.
+            vwinds (array-like): Wind speed bins.
+            speed (numpy.ndarray): Array of wind speeds.
+            dirc (numpy.ndarray): Array of wind directions (corrected).
+            direction (numpy.ndarray): Array of wind directions.
+
+        Returns:
+        numpy.ndarray: Frequency distribution of wind speeds and directions.
+        """
+        center_angle = np.linspace(0, 360, ndirections + 1)[:-1]
+        n = 180 / ndirections
+        count = np.zeros((len(center_angle), len(vwinds)))
+
+        for i in range(len(center_angle)):
+            d1 = np.mod((center_angle[i] - n), 360)
+            d2 = center_angle[i] + n
+
+            if d1 > d2:
+                cond = (dirc >= d1) | (dirc < d2)
```

Comment:
Use `or`, similarly `and`. You really don't want to use the bitwise operators here.

Reply:
Thank you for catching that. I have changed it into "or" and "and".

Reply:
I was not able to make the change work. It is showing errors.